src/Node/ModuleNode.php

```diff
@@ -33,7 +33,7 @@ class ModuleNode extends Node
     public function __construct(\Twig_NodeInterface $body, AbstractExpression $parent = null, \Twig_NodeInterface $blocks, \Twig_NodeInterface $macros, \Twig_NodeInterface $traits, $embeddedTemplates, $name, $source = '')
```

Comment:
shouldn't the typehints be updated too ?

Reply:
which ones? Twig_NodeInterface is deprecated, so we don't have a namespaced equivalent.

Reply:
ah indeed, I forgot that.